src/main/resources/spark-swagger.conf

```diff
@@ -0,0 +1,52 @@
+spark-swagger {
+
+  # UI related configs
+  theme = "MATERIAL"
+  deepLinking = false
+  displayOperationId = false
+  defaultModelsExpandDepth = 1
+  defaultModelExpandDepth = 1
+  defaultModelRendering = "model"
+  displayRequestDuration = false
+  docExpansion = "LIST"
+  filter = true
+  operationsSorter = "alpha"
+  showExtensions = false
+  showCommonExtensions = false
+  tagsSorter = "alpha"
+
+
+  # API related configs
+
+  # Location where API docs will be published
+  host = "localhost:4567"
+
+  # basepath for calling the API, if applicable, otherwise leave as "/"
+  # as in https://example.api.com/<basepath>/user/all
+  basePath = "/"
+  docPath = "/doc"
+  info {
+    description = "Open Trip Planner Middleware API"
+    title = "OTP Middleware"
+    termsOfService = ""
+    schemes = ["HTTP", "HTTPS", "WS", "WSS"]
+    project {
+      groupId = "org.opentripplanner.middleware"
+      artifactId = "otp-middleware"
+    }
+    # TODO: Enter the correct values for all properties below.
+    contact {
+      name = "IBI Group"
+      email = "sales@ibigroup.com"
```

Comment:
Probably should be a different email.

Reply:
Updated email to the IBI Group transit support.